frontend/src/static/js/v_authorship.js

```diff
@@ -316,7 +314,7 @@ window.vAuthorship = {
 
     updateSelectedFiles() {
       this.$store.commit('incrementLoadingOverlayCount', 1);
-      this.$store.commit('updateLoadingOverlayMessage', renderAuthorshipMessage);
```

Comment:
So we don't need `updateLoadingOverlayMessage` attribute right?

Reply:
I think it can be useful if you ever need to change the message. I can add a generic message used as fallback when no message is specified though.